includes/AutoLoader.php

```diff
@@ -258,6 +258,7 @@
 	'ZipDirectoryReader' => 'includes/ZipDirectoryReader.php',
 
 	# includes/actions
+	'ApprovedraftAction' => 'includes/actions/ApprovedraftAction.php',
```

Comment:
This should not be here. Please add via $wgAutoLoadClasses in the extension setup file.